deploy/pom.xml

```diff
@@ -0,0 +1,14 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>ch.ivyteam.ivy.project.demo.ci.deploy</groupId>
+  <artifactId>deploy.modules</artifactId>
+  <version>9.3.0-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <modules>
+    <module>application</module>
+<!--    <module>single-project</module> -->

```

Comment:
still needed?

Reply:
Yes, to see that this is missing and not correct on this branch. I had to implement a work around to get this stuff built. See Jenkinsfile where the addional projects are explicit built.